dev-packages/e2e-tests/test-applications/create-remix-app/app/entry.client.tsx

```diff
@@ -19,6 +19,7 @@ Sentry.init({
   replaysSessionSampleRate: 0.1, // This sets the sample rate at 10%. You may want to change it to 100% while in development and then sample at a lower rate in production.
   replaysOnErrorSampleRate: 1.0, // If you're not already sampling the entire session, change the sample rate to 100% when sampling sessions where errors occur.
   tunnel: 'http://localhost:3031/', // proxy server
+  release: 'e2e-test',
```

Comment:
potentially left over?

Reply:
No, I added this to test that this is correctly set on the INP span :)